sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -395,5 +587,163 @@ public async Task StartRecognizeCustomFormsFromUriThrowsForNonExistingContent(bo
             Assert.True(operation.HasValue);
             Assert.AreEqual(0, operation.Value.Count);
         }
+
+        private void ValidateFormPage(FormPage formPage, bool includeTextContent, int expectedPageNumber)
```

Comment:
These sanity check methods may be a bit long, but thanks to them we could spot some weird behavior, like `FormField.Pages` not taking ranges into account, or `FieldText.TextContent` being `null`.

Reply:
I like them!

Reply:
+1. Thanks Caio!!!